Updated lookup-table.md with CSV format requirements (#976)
Amplitude-Developer-Docs/amplitude-dev-center: docs/analytics/apis/lookup-tables-api.md

## Considerations

The CSV file must comply with the following requirements:

- The max file size is 100 MB and the file can't have more than 1,000,000 rows.

> - The first row should contain column names/headers.

Reviewer: Is this a requirement or a recommendation? Will the import fail if no header row is present?

Author: it's a requirement, updating..

> - The first column must correspond to the mapping property value and must contain *unique* values. Keep in mind that Lookup Tables are looking for exact matches, and are *case-sensitive*.

Reviewer: ```suggestion
- The first column must correspond to the mapping property value and must contain *unique* values. Lookup Tables search for exact matches, and are *case-sensitive*.
```
